config/config.toml

```diff
@@ -40,6 +40,13 @@ enableProcessor = true
 enableRouter = true
 enableStats = true
 
+[Http]
+ReadTimeOutInSec = 1
+ReadHeaderTimeoutInSec = 2
```

Comment:
I don't think we need read timeouts in our case, as we don't expose our server publicly. Let's keep 0 for these two as of now. It might cause issues in our case, for example, if elb/nginx opens connections in a preemptive manner, read timeout will happen before actual client request reaches our server. This is because, computation of read timeouts happen from the point of accepting a connection.